nltk/tokenize/stanford_segmenter.py

```diff
@@ -76,7 +75,7 @@ def __init__(self,
 
         # This is passed to java as the -cp option, the old version of segmenter needs slf4j.
         # The new version of stanford-segmenter-2016-10-31 doesn't need slf4j
-        # When using old version, set 'path_to_slf4j=YOUR_PATH' in the initilization of StanfordSegmenter
+        # When using old version, set path_to_slf4j='/YOUR_PATH/slf4j-api.jar' in the initilization of StanfordSegmenter
```

Comment:
Could you please move this note into the docstring that starts on the line 33. It's also better to specify concretely that versions before stanford-segmenter-2016-10-31 need to be used like this.